process/transactions/datafield/parser.go

```diff
@@ -63,6 +64,12 @@ func (odp *operationDataFieldParser) parse(dataField []byte, sender, receiver []
 		Operation: operationTransfer,
 	}
 
+	isSCDeploy := len(dataField) > 0 && isEmptyAddr(odp.pubKeyConverter, receiver)
```

Comment:
isScDeploy should be true only when the receiver is the 0000...0000 (hex) address
Otherwise, this can be mistakenly be labeled as a SC deploy tx whenever I send a transaction with something on the data field towards a new address

Reply:
right, didn't read the `isEmptyAddr` part.
Seems correct